src/Elastic.Apm/Config/AbstractConfigurationWithEnvFallbackReader.cs

```diff
@@ -73,11 +80,13 @@ internal AbstractConfigurationWithEnvFallbackReader(IApmLogger logger, string de
 		public virtual double MetricsIntervalInMilliseconds =>
 			ParseMetricsInterval(Read(KeyNames.MetricsInterval, EnvVarNames.MetricsInterval));
 
+		public bool Recording =>
+			ParseEnabled(Read(KeyNames.Enabled, EnvVarNames.Enabled));
```

Comment:
@gregkalapos

Hi, gregkalapos
Is this line mistyped?
Should it be ParseRecording(Read(KeyNames.Recording, EnvVarNames.Recording));?

Reply:
@billhong-just yes it should be, thank you for raising. I'll open a PR now to address